src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -1,70 +1,118 @@
+import moment from "moment"
 import React from "react"
-import { View } from "react-native"
+import { NavigatorIOS, View } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
 import styled from "styled-components/native"
 
 import { Icon20 } from "../Elements/Icon"
-import { Sans12, Serif16, SerifSemibold18 } from "../Elements/Typography"
+import { Sans12 } from "../Elements/Typography"
 
 import { BiddingThemeProvider } from "../Components/BiddingThemeProvider"
-import { Button } from "../Components/Button"
-import { CenteringContainer, Container } from "../Components/Containers"
-import { Divider } from "../Components/Divider"
+import { BidGhostButton, Button } from "../Components/Button"
+import { Container } from "../Components/Containers"
+import { MarkdownRenderer } from "../Components/MarkdownRenderer"
 import { Timer } from "../Components/Timer"
+import { Title } from "../Components/Title"
+
+import { BidResult_sale_artwork } from "__generated__/BidResult_sale_artwork.graphql"
+
+const SHOW_TIMER_STATUSES = ["SUCCESS", "ERROR_BID_LOW"]
 
 interface BidResultProps {
+  sale_artwork: BidResult_sale_artwork
   winning: boolean
+  status: string
   message_header?: string
   message_description_md?: string
+  navigator: NavigatorIOS
 }
 
 export class BidResult extends React.Component<BidResultProps> {
+  onPressBidAgain = () => {
+    // pushing to MaxBidScreen creates a circular relay reference but this works
+    // TODO: correct the screen transision animation
+    this.props.navigator.popToTop()
+  }
+
   render() {
+    const { live_start_at, end_at } = this.props.sale_artwork.sale
+    // non-live sale doesn't have live_start_at so bidding is open until end time
+    const timeLeftToBid = live_start_at || end_at
+    const timeLeftToBidMillis = Date.parse(timeLeftToBid) - Date.now()
     if (this.props.winning) {
       return (
         <BiddingThemeProvider>
-          <CenteringContainer>
+          <Container mt={40}>
```

Comment:
Use [`mt={6}`](https://github.com/artsy/emission/blob/develop-1.5.x/src/lib/Components/Bidding/Elements/Theme.tsx#L22) to specify `margin-bottom: 40px`.